downstream/modules/eda/proc-eda-set-up-token.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc-eda-set-up-token"]
+
+= Setting up a token to authenticate to {PlatformNameShort} Controller
+
+{ControllerNameStart} must contain a project based on a repository with certain playbooks designed to work with the {EDAName} rulebooks. 
+{ControllerNameStart} must also have corresponding job templates set up based on the playbooks in that project.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have created a user.
+* You can log in to the {EDAcontroller} Dashboard or you are added as a user in the organization.
+
+== Procedure
+
+. Navigate to the {EDAcontroller} Dashboard.
+. From the top navigation bar, select your profile.
+. Go to *User details*.
+. Select btn:[Controller Tokens] → btn:[Create controller token].
```

Comment:
I believe this is a menu selection? If so, change btn to menu as follows:
menu:Controller Tokens[Create controller token]